toolchain/check/testdata/array/array_in_place.carbon

```diff
@@ -25,7 +25,7 @@ fn G() {
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
 // CHECK:STDOUT: file {
-// CHECK:STDOUT:   package: <namespace> = namespace package, {.F = %F, .G = %G} [template]
+// CHECK:STDOUT:   package: <namespace> = namespace {.F = %F, .G = %G}, invalid [template]
```

Comment:
Hmm, should this be special cased in the formatter? Specifically, that `invalid` is used to represent a normal state here that just doesn't have any extra information to display?

Reply:
I guess I have a different interpretation here. `invalid` is the representation of InstId::Invalid being present; it's not "no extra information", it _is_ the information.

Reply:
Changed the code to elide invalid values.

Reply:
Thanks, I do somewhat prefer viewing this as a "not used" part of the instruction and not rendering it.

That said, a) you shouldn't block on this as its super easy to iterate here if we change our minds.

And b) good to double check with @zygoloid to get another view maybe?

But definitely go ahead and merge!